src/cobra/core/group.py

```diff
@@ -54,55 +58,52 @@ def __init__(self, id, name="", members=None, kind=None):
         # contains self
         self._model = None
 
-    def __len__(self):
+    def __len__(self) -> int:
+        """Get length of group."""
         return len(self._members)
 
     # read-only
     @property
-    def members(self):
+    def members(self) -> DictList:
+        """Get members of the group."""
```

Comment:
Would be great if you could please add a detailed docstring stating the arguments and returns for it.

Reply:
I'm a bit confused. There are no arguments, and it returns the members.
Do you want me to write something like
```
Returns
-------
DictList
A dictlist containing the members of the group
```